lib/procore/version.rb

```diff
@@ -1,3 +1,3 @@
 module Procore
-  VERSION = "0.8.1".freeze
+  VERSION = "0.8.2".freeze
```

Comment:
For future reference, save bumping the version for a separate Version Release branch / commit. Otherwise TravisCI breaks when the official version gets released

```
Bundler cannot continue installing procore (0.8.2).
The checksum for the downloaded `procore-0.8.2.gem` does not match the checksum
given by the server. This means the contents of the downloaded gem is different
from what was uploaded to the server, and could be a potential security issue.

To resolve this issue:
1. delete the downloaded gem located at:
`/home/circleci/my-procore/vendor/bundle/ruby/2.5.0/gems/procore-0.8.2/procore-0.8.2.gem`
2. run `bundle install`

If you wish to continue installing the downloaded gem, and are certain it does
not pose a security issue despite the mismatching checksum, do the following:
1. run `bundle config disable_checksum_validation true` to turn off checksum
verification
2. run `bundle install`

(More info: The expected SHA256 checksum was
"6ab667fb134f6d37b43151ceeef30838e0d97895d86a7a6d4ac28b3796600347", but the
checksum for the downloaded gem was
"b0deb377c068bd21a9438427b8c51cee197ac94290d42c47ec2c90e4f7e269f5".)
```

Reply:
Oh that is an excellent idea!